source/index.js

```diff
@@ -54,9 +54,7 @@
 			readdir: function(path, callback) {
 				client.getDir(endpoint, path)
 					.then(function(dirData) {
-						(callback)(null, dirData.map(function(dirEntry) {
-							return dirEntry.basename;
-						}));
+						(callback)(null, dirData);
```

Comment:
I'm trying to keep webdav-fs as close to the expected output of native `fs` methods as possible: https://nodejs.org/api/fs.html#fs_fs_readdir_path_options_callback

> Reads the contents of a directory. The callback gets two arguments (err, files) where files is an array of the names of the files in the directory excluding '.' and '..'.

So passing back just the dirData is incorrect, and this is confirmed in the tests: https://github.com/perry-mitchell/webdav-fs/blob/master/test/specs/readdirTests.js#L32